### PR TITLE
(WIP) Improve site

### PR DESCRIPTION
**Pull request by zanedb**
Work in progress, fixing a few things.

### CODE REVIEW

File: src/components/EventCard.js [hunk 1]
```diff
@@ -137,6 +132,7 @@ export default ({
     <EventCard bg={banner}>
       <AssociatedSeal mlhAssociated={mlh} />
       <LogoContainer>
+        {console.log(id, logo, banner, website, name)}
```
Comment (lachlanjc) [inline src/components/EventCard.js]: ⚠️

File: src/components/EventCard.js [hunk 2]
```diff
@@ -49,37 +49,32 @@ const EventCard = Flex.withComponent(Tilt).extend.attrs({
...
   ml: -theme.space[4],
   mt: -theme.space[3] + 5,
   mb: 1,
   children: props => (
-    <MLHLogo
-      src="/mlh-logo-grayscale.svg"
-      alt="MLH logo"
-      mlhAssociated={props.mlhAssociated}
-    />
+    <>
```
Comment (lachlanjc) [inline src/components/EventCard.js]: Can we remove this fragment?

File: src/components/EventCard.js (hunk 1 above)
Comment (zanedb) [reply src/components/EventCard.js]: Yup, this was temporary. needed it to deploy :)

File: src/components/EventCard.js (hunk 2 above)
Comment (zanedb) [reply src/components/EventCard.js]: Would like to, but unfortunately it throws a ESLint error. see: https://stackoverflow.com/a/9534132

**Comment by maxwofford** [comment]
Closing due to inactivity. Feel free to reopen if you think it should be

**Pull request closed without merging**